extensions/amp-subscriptions/0.1/subscription-platform.js

```diff
@@ -73,6 +73,12 @@ export class SubscriptionPlatform {
    * @returns {number}
    */
   getBaseScore() {}
+
+  /**
+   * Decorate the DomNode according to your platform
+   * @param {Element} unusedElement
+   */
+  decorateUI(unusedElement) {}
```

Comment:
Same here: let's call it `attachAction(element, action, opt_decorate)` or similar. Basically, it needs to mimic whatever subscriptions service calls it.